lib/Controller/Layout.php

```diff
@@ -1542,6 +1542,9 @@ public function grid(Request $request, Response $response)
                         // Augment with deletable flag
                         $widget->setUnmatchedProperty('isDeletable', $this->getUser()->checkDeleteable($widget));
 
+                        // Augment with viewable flag
+                        $widget->setUnmatchedProperty('isViewable', $this->getUser()->checkViewable($widget));
```

Comment:
These are actually always viewable because they are returned by the factory which does a view check by default - so we could just set these to true

Reply:
Sorry, I mean the playlist one!